client/src/application/containers/utils/navbar/index.tsx

```diff
@@ -77,6 +79,21 @@ class Navbar extends React.Component<IProps, IState> {
         super(props);
     }
 
+    state = {
+        redirect: false
+    }
+
+    setRedirect = () => {
+        this.setState({
+            redirect: true
+        })
+    }
+    renderRedirect = () => {
+        if (this.state.redirect) {
+            return <Redirect to='/' />
+        }
+    }
+
```

Comment:
This part should become useless if you use `this.props.history.push` as mentioned in the other comment :)